portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -116,7 +116,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     try {
       url = appendQuery(url, "commentSort=" + sortOrder).toString();
       url = appendQuery(url, "numComments=" + numDisplay).toString();
-    } catch (URISyntaxException e) {}
+    } catch (URISyntaxException e) {
+      System.out.println("Query string invalid!");
+    }
```

Comment:
[Optional] This is the kind of thing that could be factored out into its own PR as we discussed in our 1:1.  If you want to try getting comfortable with rebasing/splitting changes this would be a good case for you to try because it is simple (all the changes that need to be extracted to another branch are isolated to a single file).